.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "eamodio.gitlens",
+    "esbenp.prettier-vscode",
+    "jpoissonnier.vscode-styled-components",
+    "rvest.vs-code-prettier-eslint"
```

Comment:
How come prettier isn't already enabled through vscode-eslint? What does this add?